routes/admin/not-authenticated/account-register/page.js

```diff
@@ -27,41 +28,116 @@ class RegisterPage extends Component {
+              }
+            />
             </FormGroup>
+
             <FormGroup className='col-6' controlId='lastNameId' {...lastName}>
-              <ControlLabel>Sobrenome</ControlLabel>
-              <FormControl type='text' placeholder='Sobrenome' />
+              <ControlLabel>
+                <FormattedMessage
+                  id='page--account-register.form.lastname.label'
+                  defaultMessage='Sobrenome'
+                />
+              </ControlLabel>
+
+                <FormControl
+                  type='text'
+                  placeholder={
+                    intl.formatMessage({
+                      id: 'page--account-register.form.lastname.placeholder',
+                      defaultMessage: 'Sobrenome'
+                    })
+                  }
+                />
             </FormGroup>
           </div>
+
           <FormGroup controlId='emailId' {...email}>
-            <ControlLabel>E-mail</ControlLabel>
-            <FormControl type='email' placeholder='exemplo@email.com.br' />
+            <ControlLabel>
+              <FormattedMessage
+                id='page--account-register.form.email.label'
+                defaultMessage='E-mail'
+              />
+            </ControlLabel>
+
+            <FormControl
+              type='email'
+              placeholder={
+                intl.formatMessage({
+                  id: 'page--account-register.form.email.placeholder',
+                  defaultMessage: 'exemplo@email.com.br'
+                })
+              }
+            />
           </FormGroup>
           <FormGroup controlId='passwordId' {...password}>
-            <ControlLabel>Senha</ControlLabel>
-            <FormControl type='password' placeholder='********' />
-          </FormGroup>
+            <ControlLabel>
+              <FormattedMessage
+                id= 'page--account-register.form.password.label'
+                defaultMessage= 'Senha'
+              />
+              </ControlLabel>
+              <FormControl type='password' placeholder='********' />
+            </FormGroup>
           <FormGroup controlId='password2Id' {...password2}>
-            <ControlLabel>Confirme sua senha</ControlLabel>
+            <ControlLabel>
+              <FormattedMessage
+                id='page--account-register.form.password-confirm.label'
+                defaultMessage='Confirme sua senha'
+              />
+            </ControlLabel>
             <FormControl type='password' placeholder='********' />
           </FormGroup>
           <Button type='submit' className='btn caps col-12 p2 white rounded-bottom bg-pagenta'>
-            {formProps.submitting ? 'Salvando...' : 'Criar conta'}
+            {formProps.submitting ?
+            (
+              <FormattedMessage
+                id='page--account-register.form.submit-button.saving'
+                defaultMessage='Salvando...'
+              />
+            )
+               : <FormattedMessage
```

Comment:
if the first condition of ternary condition uses parenthesis, the second one must have parenthesis too..